web/themes/new_weather_theme/templates/partials/daily-high-low.html.twig

```diff
@@ -1,9 +1,23 @@
+{% set temps = periods | map(p => p.temperature) %}
+{% set low = min(temps) %}
+{% set high = max(temps) %}
 <div class="daily-high-low display-flex flex-column flex-align-center padding-x-4px margin-right-2">
   <span class="display-block text-bold margin-bottom-05 font-mono-3xs font-family-mono text-base-dark text-bold">
-    {{period.isDaytime ? "High" | t | upper : "Low" | t | upper}}
+    {{ "High" | t | upper }}
```

Comment:
I wonder if we should _**not**_ uppercase it, and use CSS to get the uppercase styling. All caps could be unpleasant with a screen reader, and possibly that's why it was reading L-O-W instead of saying the word "low"?

Reply:
Definitely, will fix and push